packages/server/modules/gatekeeper/index.ts

```diff
@@ -73,10 +82,59 @@ const scheduleWorkspaceTrialEmails = ({
 }: {
   scheduleExecution: ScheduleExecution
 }) => {
+  const sendWorkspaceTrialEmail = sendWorkspaceTrialExpiresEmailFactory({
+    getServerInfo: getServerInfoFactory({ db }),
+    getUserEmails: findEmailsByUserIdFactory({ db }),
+    getWorkspaceCollaborators: getWorkspaceCollaboratorsFactory({ db }),
+    sendEmail,
+    renderEmail
+  })
   // TODO: make this a daily thing
-  const cronExpression = '*/5 * * * *'
+  // const cronExpression = '*/5 * * * * *'
+  // every day at noon
+  const cronExpression = '0 12 * * *'
   return scheduleExecution(cronExpression, 'WorkspaceTrialEmails', async () => {
-    // await manageSubscriptionDownscale()
+    const getWorkspacesByPlanAge = getWorkspacesByPlanAgeFactory({ db })
+    const trialValidForDays = 31
+    const trialWorkspacesExpireIn14Days = await getWorkspacesByPlanAge({
+      daysTillExpiry: 14,
+      planValidFor: trialValidForDays,
+      plan: 'starter',
+      status: 'trial'
+    })
+    if (trialWorkspacesExpireIn14Days.length) {
```

Comment:
@alemagio I think we can remove the 14 days email for now. I initially wanted it but now I think it's a little too much to send it this long time before the trial ends. 3 days before and on the same day is enough for now.

Maybe we will send another type of email 14 days before just to check in with the user, but that can be a separate task. Additionally we'll also be adding more automated emails _after_ the trial has expired to try and reactive the workspace. But also a separate task.